src/test/resources/features/comp/ComplaintClosed.feature

```diff
@@ -0,0 +1,23 @@
+@ComplaintClosed @COMP
+Feature: Complaint Closed
+
+  Background:
+    Given I am logged into "DECS" as user "COMP_USER"
+    And I create a "COMP" case and move it to the "Complaint Closed" stage
+    And I load and claim the current case
+
+#    HOCS-2716
+  @COMPWorkflow @COMPRegression
```

Comment:
Add the @COMPWorkflow @COMPRegression to the top of the feature and not required for each scenario